how-to-guides/feegrant-for-blobs.md

```diff
@@ -198,3 +204,85 @@ celestia-appd query bank balances $GRANTER_ADDRESS \
 
 This output will show the remaining balance after fees have been deducted,
 confirming that the FeeGrant module is working as intended.
+
+## Optional: Using FeeGrant with curl commands
+
+To grant fee allowances using curl commands, which can be useful for automation
+or when you need more direct control over the RPC calls, follow these steps.
+
+First, make sure you have `curl` and `jq` installed on your system.
+
+1. Start your local light node for Mocha testnet:
+
+```bash
+celestia light start --p2p.network mocha --core.ip rpc-mocha.pops.one
+```
+
+2. In a new terminal, export the authentication token:
+
+```bash
+CELESTIA_NODE_AUTH_TOKEN=$(celestia light auth admin --p2p.network mocha)
```

Comment:
I appreciate that you used a variable and it avoids needing an example token here!